docs/source/install/system_requirements.rst

```diff
@@ -30,3 +30,5 @@ for the best experience while testing or deploying |st2|:
 |  * 10GB storage                      | * 40GB storage                    |
 |  * Recommended EC2: **t2.medium**    | * Recommended EC2: **m4.xlarge**  |
 +--------------------------------------+-----------------------------------+
+
+By default, |st2| and related services use these ports: 80, 443, 27017, 4369, 5432, 5672, 25672, 9100, 9101 and 9102. If any other services are currently using these ports, |st2| may fail to install or run correctly.
```

Comment:
I liked "group by service" approach in previous version, ex: `rabbitmq (4369, 5672, 25672)`.
Gives better picture, comparing to just listing all the ports.